benchmark/src/main/scala/code/winitzki/benchmark/Benchmarks9.scala

```diff
@@ -33,7 +33,7 @@ object Benchmarks9 {
 
     val tp = new FixedPool(threads)
```

Comment:
In that case, I'd have to create a new threadpool for each benchmark outside. (I don't want different benchmarks to influence each other.) Maybe this is better.